EventListener/ViewResponseListener.php

```diff
@@ -58,55 +60,48 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             return false;
         }
 
-        /** @var \FOS\RestBundle\Controller\Annotations\View $configuration */
-        $configuration = $request->attributes->get('_view');
+        $configuration = $request->attributes->get('_template');
 
         $view = $event->getControllerResult();
-        $customViewDefined = true;
         if (!$view instanceof View) {
-            if (!$configuration && !$this->forceView) {
+            if (!$configuration instanceof ViewAnnotation && !$this->forceView) {
```

Comment:
`$configuration` will now never be an instance of `ViewAnnotation` if our listener is executed before the one from the SensioFrameworkExtraBundle, will it?